components/menu/index.tsx

```diff
@@ -24,7 +24,7 @@ const AccountButton = dynamic(() => import("../account/AccountButton"), {
 const TopBar = () => {
   return (
     <div
-      className={`w-full py-3.5 sticky top-0 z-40 transition-all duration-300 bg-black h-topbar-height`}
+      className={`w-full py-3.5 fixed top-0 z-40 transition-all duration-300 bg-black h-topbar-height`}
```

Comment:
This seems to be unrelated to the css change but I'm posting here since it's relevant: previously the scroll was disabled on mobile so that when the hamburger menu was open the user couldn't scroll up/down. Perhaps this could be opened as a separate issue.

Reply:
i think we can leave it as is and open an issue for it if deemed necessary.